src/helpers.test.ts

```diff
@@ -48,6 +49,7 @@ test('getBaseUrl', () => {
   expect(PLATFORMS.map((platform) => {
     return getBaseUrl(platform, 'v0.1.0_8880');
   })).toEqual([
+    'https://uploader.codecov.io/v0.1.0_8880/aarch64/codecov',
```

Comment:
https://uploader.codecov.io/v0.5.0/aarch64/codecov is currently the only version available for aarch64